recipes/default.rb

```diff
@@ -32,10 +32,12 @@
       action :add
     end
   end
+  repo_url = "http://yum.mackerel.io/centos/$basearch"
+  repo_url = "http://yum.mackerel.io/amznlinux/$releasever/$basearch" if platform?('amazon')
```

Comment:
I don't check `platform_version` like below:

```ruby
if platform?('amazon') && node['platform_version'] == '2016.03'
  ...
```

Because I understand the new repo is required only Amazon Linux 2016.03, but it can be used for Amazon Linux 2015.09 and older Amazon Linux.

This is based on below description.

> 正式サポートに際し、Amazon Linux をご利用のお客様は新しいリポジトリをご利用いただけます。 そのため、従来のリポジトリをご利用の場合はリポジトリの切替が必要となります。 ただし、Amazon Linux AMI 2016.03より前のAmazon Linuxは動作影響が無いため、リポジトリの切替は必須ではありません。
https://mackerel.io/ja/blog/entry/2016/03/28/195614

If it has to check the version, please let me know. I'll change it.